addons/mail/static/src/chatter/web/chatter.xml

```diff
@@ -44,7 +44,7 @@
                 <div class="o-mail-Followers d-flex me-1">
                     <Dropdown position="'bottom-end'" menuClass="'o-mail-Followers-dropdown d-flex flex-column'" state="followerListDropdown">
                         <button t-att-class="'o-mail-Followers-button btn btn-link d-flex align-items-center text-action px-1 ' + (props.compactHeight ? '' : 'my-2')" t-att-disabled="isDisabled" t-att-title="followerButtonLabel">
-                            <i class="fa fa-user-o me-1" role="img"/>
+                            <i class="fa me-1" role="img" t-att-class="{ 'fa-bell': state.thread.selfFollower, 'fa-bell-o': !state.thread.selfFollower }"/>
```

Comment:
Should use `fa-fw` for giving more guarantee that the button stays the same size thanks to constant-sized icon.